app/main.py

```
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+
+    if len(split_command) != 3 or split_command[0] != "cp":
+        return
```

Consider providing feedback to the user when the command is invalid. Currently, the function silently returns if the command is not in the expected format. Adding a print statement or logging could help in debugging or user feedback.